Web_Development/FrontEnd/CSS Image Reflection/CSS_Image_Reflection.md

```diff
@@ -0,0 +1,62 @@
+# CSS Image Reflection
+The **box-reflect** property is used to reflect an image in CSS.<br>
+**box-reflect** can hold the following values:
```

Comment:
- Use one line space after and before the headings.
- Use one line space after and before paragraphs.
- Don't use `<br>`.
- Give code inside `inline code` not using **bold**.